astropy/io/fits/tests/test_table.py

```diff
@@ -3335,6 +3335,59 @@ def test_multidim_VLA_tables(self):
                 hdus[1].data["test"][1], np.array([[0.0, 1.0, 2.0], [3.0, 4.0, 5.0]])
             )
 
+    def test_heterogeneous_VLA_tables(self):
+        """
+        Check the behaviour of heterogeneous VLF object.
```

Comment:
I cannot tell if this is a typo or VLF is something else.
```suggestion
        Check the behaviour of heterogeneous VLA object.
```

Reply:
_VLF is the Variable Length Field defined in `columns.py` , VLA is for Variable Length Array ... I am not sure when one is better than the other... in that case I think you are right.